Assets/Scripts/Characters/Humans/Hero.cs.meta

```diff
@@ -1,11 +1,10 @@
 fileFormatVersion: 2
 guid: c2ade5059944e77468ce66f0f1fb36c9
-timeCreated: 1582403864
-licenseType: Free
 MonoImporter:
+  externalObjects: {}
   serializedVersion: 2
   defaultReferences: []
-  executionOrder: 0
+  executionOrder: 5
```

Comment:
This part is important, this fixes the hook target offset by ensuring that the camera position is correct, since with default timing, the returned camera positions is ~1.4 units too high.